### PR TITLE
Grammar and slight ordering changes

### PR DESCRIPTION
**Pull request by Xmasreturns**
Updated structs.md in the book

### CODE REVIEW

File: src/doc/book/structs.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ let origin_x = 0;
 let origin_y = 0;
 ```
 
-A `struct` lets us combine these two into a single, unified datatype:
+A `struct` lets us combine these two into a single, unified datatype with `x`
+and `y` as traits:
```
Comment (steveklabnik) [inline src/doc/book/structs.md]: x and y are not traits here

File: src/doc/book/structs.md [hunk 2]
```diff
@@ -18,7 +19,7 @@ struct Point {
 }
 
 fn main() {
-    let origin = Point { x: 0, y: 0 }; // origin: Point
+    let origin = Point { x: 0, y: 0 }; // create an instance
```
Comment (steveklabnik) [inline src/doc/book/structs.md]: This is a convention we've been using throughout the starting bits of the book, to annotate the types of things.

**Comment by Xmasreturns** [comment]
Thanks for the heads up on the `name: Type` convention, I must've missed that while reading. Also, good catch on x/y - I definitely meant field labels there, sorry!

**Pull request merged**